scripts/indexApi.js

```diff
@@ -25,3 +25,26 @@ applist.find = function (id) {
       return apps.find(findApp)
     })
 }
+applist.add = function (app) {
```

Comment:
@gr2m @titay2 By accident, I added the applist.add twice :D Thank you for the feedback and fixing that part :) Yeahhyyy, happy to have the best teamwork! :D